yarn.lock

```diff
@@ -146,12 +146,17 @@
   resolved "https://registry.yarnpkg.com/@foxglove/message-definition/-/message-definition-0.2.0.tgz#e31922e58e57d224717bcfbd20d3cbaad709dc6b"
   integrity sha512-IQHIGCvBZR8GIua9nEpS+hsMF3gm1bfbrrnjG0rgtcFBWiNuKbzx4vIP8OIwDC+8wtwcFdfJhf4Vp5TPFiUUcQ==
 
-"@foxglove/rosmsg@^4.0.0":
-  version "4.0.0"
-  resolved "https://registry.yarnpkg.com/@foxglove/rosmsg/-/rosmsg-4.0.0.tgz#b0f1b57db4bfe2ea00f1d5904caeedb2d13bccb0"
-  integrity sha512-MuTAcA4U+wzlhUIHYgfBrjTDE/kzukWmhyI8suKgmjlnVjrpyzDNiMYsV5lU0KTW6noGKtqu46ZtxWSEWsOsEg==
+"@foxglove/message-definition@^0.3.1":

```

Comment:
@achim-k how do we avoid getting two versions of `@foxglove/message-definition` in this package's dependencies? I think our packages should generally avoid this if we can